web/src/app/users/UserContactMethodCreateDialog.js

```diff
@@ -22,16 +22,40 @@ export default function UserContactMethodCreateDialog(props) {
     name: '',
     type: 'SMS',
     value: '',
+    countryCode: 'US', // default
+    countryName: 'United States',
   })
 
+  useEffect(() => {
+    async function fetchIpLocale() {
+      const response = await fetch('http://ip-api.com/json')
```

Comment:
We should do this once globally rather than every time the dialog opens. Also, is using it this way allowed by their ToS?